src/stp/new_tactics/KickAtPos.cpp

```diff
@@ -40,14 +40,16 @@ StpInfo KickAtPos::calculateInfoForSkill(StpInfo const &info) noexcept {
 
     // When rotating, we need to dribble to keep the ball, but when kicking we don't
     if (skills.current_num() == 0) {
```

Comment:
Suggestion: if robot is not facing the shooting position, dribble. It makes more sense logic-wise

Reply:
Done